ow_sim_tests/test/sample_collection.py

```diff
@@ -304,36 +306,36 @@ def test_04_discard(self):
   """
   Grind a new trench in a different location on the terrain.
   """
-  def test_05_grind(self):
-    grind_result = test_arm_action(self,
+  def test_05_task_grind(self):
+    task_grind_result = test_arm_action(self,
       'TaskGrind', owl_msgs.msg.TaskGrindAction,
-      owl_msgs.msg.GrindGoal(
+      owl_msgs.msg.TaskGrindGoal(
         x_start         = 1.65,
         y_start         = 0.5,
         depth           = 0.05,
         length          = 0.6,
         parallel        = True,
         ground_position = GROUND_POSITION
       ),
-      TEST_NAME, "test_05_grind",
+      TEST_NAME, "test_05_task_grind",
       condition_check = self._assert_regolith_not_present
     )
 
   """
   Dig for more material in the new trench, so we can then test delivery into
   the sample dock.
   """
-  def test_06_dig_circular(self):
-    dig_circular_result = test_arm_action(self,
+  def test_06_task_scoop_circular(self):
+    task_scoop_circular_result = test_arm_action(self,
       'TaskScoopCircular', owl_msgs.msg.TaskScoopCircularAction,
       owl_msgs.msg.TaskScoopCircularGoal(
         frame    = 0,
         relative = False,
-        point    = Point(1.65, 0.5, 0.01),
+        point    = Point(1.65, 0.5, ow_lander.constants.DEFAULT_GROUND_HEIGHT),
         depth    = 0.01,
-        parallel = True
+        parallel = True,
```

Comment:
```suggestion
        parallel = True
```

Reply:
Found similar problem in arm_check_actions. All of them are adjusted.